<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuSolver</class>
 <widget class="QMainWindow" name="SudokuSolver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SudokuSolver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="SudokuBoard">
      <item row="3" column="3">
       <widget class="QSpinBox" name="spinBox_33">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinBox_04">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_03">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QSpinBox" name="spinBox_08">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spinBox_05">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="spinBox_30">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSpinBox" name="spinBox_82">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="spinBox_81">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_12">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinBox_32">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QSpinBox" name="spinBox_47">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QSpinBox" name="spinBox_48">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QSpinBox" name="spinBox_56">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="QSpinBox" name="spinBox_57">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QSpinBox" name="spinBox_66">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QSpinBox" name="spinBox_74">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="7">
       <widget class="QSpinBox" name="spinBox_77">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="QSpinBox" name="spinBox_67">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="spinBox_73">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="8">
       <widget class="QSpinBox" name="spinBox_68">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QSpinBox" name="spinBox_84">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QSpinBox" name="spinBox_75">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QSpinBox" name="spinBox_76">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QSpinBox" name="spinBox_83">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QSpinBox" name="spinBox_65">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="8">
       <widget class="QSpinBox" name="spinBox_78">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QSpinBox" name="spinBox_36">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="spinBox_24">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QSpinBox" name="spinBox_44">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="spinBox_50">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_11">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="spinBox_63">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spinBox_15">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QSpinBox" name="spinBox_70">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QSpinBox" name="spinBox_60">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="spinBox_25">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="spinBox_14">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QSpinBox" name="spinBox_26">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="spinBox_16">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="spinBox_06">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QSpinBox" name="spinBox_40">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QSpinBox" name="spinBox_80">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QSpinBox" name="spinBox_37">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBox_51">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_31">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox_41">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_21">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="spinBox_53">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QSpinBox" name="spinBox_55">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QSpinBox" name="spinBox_54">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSpinBox" name="spinBox_17">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QSpinBox" name="spinBox_27">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QSpinBox" name="spinBox_18">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QSpinBox" name="spinBox_28">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QSpinBox" name="spinBox_38">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="spinBox_71">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinBox_61">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBox_00">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox_10">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_01">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="spinBox_20">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_02">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QSpinBox" name="spinBox_07">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="spinBox_72">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="spinBox_62">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QSpinBox" name="spinBox_85">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QSpinBox" name="spinBox_86">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBox_42">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_13">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox_23">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="spinBox_52">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="8">
       <widget class="QSpinBox" name="spinBox_88">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_22">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="8" column="7">
       <widget class="QSpinBox" name="spinBox_87">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QSpinBox" name="spinBox_46">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QSpinBox" name="spinBox_45">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QSpinBox" name="spinBox_35">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="spinBox_34">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QSpinBox" name="spinBox_64">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBox_43">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="5" column="8">
       <widget class="QSpinBox" name="spinBox_58">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="solveButton">
        <property name="text">
         <string>Solve Sudoku Puzzle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="displayAnswersButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Display Answer</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eraseButton">
        <property name="text">
         <string>Clear Sudoku Grid</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
